rabbitmq/datadog_checks/rabbitmq/rabbitmq.py

```diff
@@ -200,6 +200,50 @@ def _get_config(self, instance):
 
         return base_url, max_detailed, specified, custom_tags, suppress_warning
 
+    def _collect_metadata(self, base_url):
+        # Versioning in RabbitMQ seems to be standard semantic versioning.
+        # https://www.rabbitmq.com/changelog.html
+
+        # The /overview endpoint from the RabbitMQ status API outputs
+        # JSON in the following format which includes versioning information
+        # more information here: https://pulse.mozilla.org/doc/stats.html
+        '''
+        {
```

Comment:
You can use the example payload as a test fixture  rather than putting it here in a comment